package.json

```diff
@@ -21,28 +21,28 @@
   },
   "homepage": "https://github.com/Automattic/eslint-config-wpvip#readme",
   "dependencies": {
-    "eslint-config-wpcalypso": "^4.0.0",
+    "eslint-config-wpcalypso": "^6.1.0",
     "eslint-loader": "^3.0.0",
     "webpack": "^4.21.0"
   },
   "peerDependencies": {
     "@babel/eslint-parser": "^7.13.14",
-    "eslint": "^5.0.0 || ^6.0.0",
-    "eslint-plugin-flowtype": "^3.0.0 || ^4.0.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint": "^7.25.0",
+    "eslint-plugin-flowtype": "^5.7.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-no-async-foreach": "^0.1.1",
-    "eslint-plugin-react": "^7.10.0",
-    "eslint-plugin-wpcalypso": "^4.0.1",
-    "eslint-plugin-json": "^1.2.1 || ^2.0.0",
```

Comment:
@nejclovrencic I'm not sure we can change these peerDependencies - IIRC, some of these come from the upstream eslint-config-wpcalypso package and some are for compatibility with the various repos that use this config plugin. We'll need to ensure both of those are compatible with these changes.

Reply:
Yes, we talked with @robersongomes last week, and we'll need to update those for npm v7, but apparently latest wpcalypso is not published to npm anymore.